spinalcordtoolbox/reports/assets/_assets/js/main.js

```diff
@@ -39,26 +43,23 @@ $(document).ready(function(){
     ele.style.marginLeft = "10px";
   }
 
-  function newScroll(newRow)
-  {
-    var rowTop = newRow.position().top;
-    var rowBottom = rowTop + newRow.height();
-    var $table = $('.fixed-table-body'); // store instead of calling twice
-    var tableHeight = $table.height();
-    var currentScroll = $table.scrollTop();
-    
-    if (rowTop < 0)
-    {
-        // scroll up
-        $('.fixed-table-body').scrollTop(currentScroll + rowTop - 20);
-    }
-    else if (rowBottom  > tableHeight)
-    {
-        // scroll down
-        var scrollAmount = rowBottom - tableHeight;
-        $('.fixed-table-body').scrollTop(currentScroll + scrollAmount + 20);
-    }
-    
+  function newScroll(newRow) {
+    if (newRow && newRow.length > 0) {
```

Comment:
i've added this because of an error in the developer tools of the browser-- i do not have the time to further dig, sorry

Reply:
I just tested this, and you are correct that an error is thrown:

```javascript
TypeError: newRow.position() is undefined
```

Just to document the error, I found that the error occurs when pressing up/down/`j`/`k` when first loading the page (i.e. no row has been highlighted yet):

This error occurs because, on first page load, there is no active row in the table. So, when we call this line:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/3ad3a615bcde6d8c4c7c2f95272af23f7906dac6/spinalcordtoolbox/reports/assets/_assets/js/main.js#L122

`obj` will be `null`.

Your solution (skipping the internals of `newScroll` if `newRow` is null/has zero length) does indeed work. That said, I wanted to get to the root of the issue. So, what I did was make sure that `obj` is properly set in the cases where there is no active row -- see https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4317/commits/cfdd7fcfc0b25d677625767aa3396daefbe79493.